cve_bin_tool/version_scanner.py

```diff
@@ -129,12 +131,18 @@ def scan_file(self, filename):
         # parse binary file's strings
         if inpath("strings"):
             # use "strings" on system if available (for performance)
-            o = subprocess.check_output(["strings", filename])
-            lines = o.decode("utf-8").splitlines()
+            s = subprocess.check_output(["strings", filename])
```

Comment:
Maybe don't change this from o to s.  We'll need to find another solution, maybe a separate block.

Reply:
Should start with changing both s and o to things that are more clear, or put them on single lines so we don't have the temp varibles at all.